python/py-huggingface-transformers/Portfile

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+PortGroup           github 1.0
+
+github.setup        huggingface transformers 2.1.1 v
+revision            0
+name                py-huggingface-transformers
+
+license             Apache-2
+maintainers         nomaintainer
+platforms           darwin
+
+description         State-of-the-art Natural Language Processing for\
+                    TensorFlow 2.0 and PyTorch
+long_description \
+    🤗 Transformers (formerly known as pytorch-transformers and\
+    pytorch-pretrained-bert) provides state-of-the-art general-purpose\
+    architectures (BERT, GPT-2, RoBERTa, XLM, DistilBert, XLNet,\
+    CTRL...) for Natural Language Understanding (NLU) and Natural\
+    Language Generation (NLG) with over 32+ pretrained models in 100+\
+    languages and deep interoperability between TensorFlow 2.0 and\
+    PyTorch.
+
+homepage            https://huggingface.co/transformers/
+
+python.versions     27 37 38
```

Comment:
Again, I suspect 2.7 should no longer be added to brand new ports as it's going to be out of support in less than two months.

Reply:
Done thanks.